examples/apidoc/V5/P2P/upload-chat-file.js

```diff
@@ -0,0 +1,24 @@
+const { RestClientV5 } = require('bybit-api');
+const fs = require('fs');
+const FormData = require('form-data');
```

Comment:
Btw apparently FormData is already available in node as of Node 18. No need for import/require:
https://developer.mozilla.org/en-US/docs/Web/API/FormData#browser_compatibility